src/Akeneo/Crowdin/Api/AddDirectory.php

```diff
@@ -44,11 +44,9 @@ public function execute()
             $parameters['branch'] = $this->branch;
         }
 
-        $request  = $this->client->getHttpClient()->post($path, [], $parameters);
+        $response = $this->client->getHttpClient()->post($path, $parameters);
```

Comment:
Sorry, I was maybe not clear in my explaination :)

In fact any call to "$this->client->getHttpClient()->post(" needs to be changed to work with GuzzleHttp.

There are 2 cases described here http://docs.guzzlephp.org/en/latest/quickstart.html#post-form-requests

First case is to send form data with files, used by classes 'AddFile', 'UpdateFile' and 'UploadTranslation', it can be changed as we did for the 'UpdateFile' by using 'multipart' and the 'fopen'.

The second case for all other classes which post raw data as 'AddDirectory' is to do the following (I tried it work well):
```
-        $response = $this->client->getHttpClient()->post($path, $parameters);
+        $data = ['form_params' => $parameters];
+        $response = $this->client->getHttpClient()->post($path, $data);
```